docs/tutorials/BUILD

```diff
@@ -23,8 +23,8 @@ py_binary(
   data = [":notebooks_folder",
           "//bark:generate_core",
           "//bark/runtime/tests:xodr_data",
+          "//examples/mcts_rl/database:database",
           "//examples:params",
-          "@benchmark_database//data/database1:database1",
```

Comment:
you can not remove this line since then the other notebooks wont work anymore. Please ensure that all other notebooks still work